datastore-api/app/core/config.py

```diff
@@ -19,6 +19,7 @@ class Settings(BaseSettings):
     MONGO_HOST: str = ""
     MONGO_PORT: int = 27017
     MONGO_SERVER_SELECTION_TIMEOUT_MS: int = 3000
+    BING_KEY = ""
```

Comment:
The `.env` file is not available in the image, so it cannot load the variables via the Config class. Therefore, please load the env variables with this syntax:
```python3
BING_KEY = Field(..., env="BING_KEY")
```
This will check the environment for a variable called `BING_KEY` and load the value.

Reply:
@timbmg Done!